console/console-init/ui/src/graphql-module/queries/Connection.ts

```diff
@@ -349,7 +349,7 @@ const CONNECTION_LINKS_FILTER = (
         filterForLink += "`$.spec.address` = '" + filterAddresseValue + "'";
       else
         filterForLink +=
-          "`$.spec.Address` LIKE '" + filterAddresseValue + "%' ";
+          "`$.spec.address` LIKE '" + filterAddresseValue + "%' ";
```

Comment:
There seems to be a typo in the variable filterAddresseValue. It should instead have been 'filterAddressValue'